### PR TITLE
US88463-reposition of editnavbar

### PR DESCRIPTION
**Pull request by edwardg8**

### CODE REVIEW

File: sass/navigation/main.scss [hunk 1]
```diff
@@ -11,6 +12,7 @@
 	display: flex;
 	height: calc(1rem + 40px);
 	flex-wrap: nowrap;
+	margin-right: 45px;
```
Comment (dlockhart) [inline sass/navigation/main.scss]: What's this for? Could you provide some reasoning behind the change or some before/after screenshots?

Comment (edwardg8) [inline sass/navigation/main.scss]: We want to re-position the edit navbar menu for text-based nav (details refer to `US88463` - has screenshot of the changes). Earlier we have the edit menu above more menu, now we want to move it to the right side of more menu. The 45px is reserving the space for this edit navbar menu (35px width of menu +10px width space between the last link and the edit menu

Comment (edwardg8) [inline sass/navigation/main.scss]: Before:
image

After:
image

Comment (dlockhart) [inline sass/navigation/main.scss]: Hmm. So for people who don't have permission to see the menu (all students), there'll be an empty space to the right. I'd recommend putting a classname somewhere when it renders to indicate that the menu is going to be present, and then only apply the margin when that class is there.

Comment (edwardg8) [inline sass/navigation/main.scss]: Thank you for catching that, make the change right now.

Comment (edwardg8) [inline sass/navigation/main.scss]: I made changes from LMS, linked the BSI changes here to LMS PR

File: sass/navigation/main.scss [hunk 2]
```diff
@@ -23,6 +24,10 @@
 	}
 }
 
+.d2l-navigation-s-main-wrapper[has-edit-menu] {
+	margin-right: 45px;
```
Comment (dlockhart) [inline sass/navigation/main.scss]: What about RTL?

Comment (edwardg8) [inline sass/navigation/main.scss]: Wow, the eidtmenu for both image-based or text-based, there isn't RTL considered. Thanks for pointing it out. I'm going to test and address all RTL problems here

**Pull request merged**